.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - lua5.1
+    - luarocks
+
+install:
+  - luarocks --local install luafilesystem
+  - luarocks --local install luacheck
+  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc --no-check-certificate https://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Server/.luacheckrc
```

Comment:
I don't have any experience with this, but you're trying to use a file called `luacheckrc` while the file you've added is called `luacheckrc_plugin`. Is this correct?